modules/openapi/test/resources/foo.json

```diff
@@ -34,6 +34,10 @@
     },
     "/hello/{name}/{ts}": {
       "get": {
+        "externalDocs": {
+          "description": "APIHomepage2",
```

Comment:
is it expected that we strip all of the spaces here ?

Reply:
No.. however, I can't yet figure out where that is happening. Seems to be inside of the aws-smithy-openapi stuff. I will investigate further